website/docs/r/monitor_diagnostic_setting.html.markdown

```diff
@@ -63,8 +63,6 @@ The following arguments are supported:
 
 * `eventhub_name` - (Optional) Specifies the name of the Event Hub where Diagnostics Data should be sent. Changing this forces a new resource to be created.
 
--> **NOTE:** If this isn't specified then the default Event Hub will be used.
```

Comment:
Can we revert this change on the document as this PR aims to add examples?

Reply:
added back the original comments